pdfplumber/cli.py

```diff
@@ -23,13 +24,16 @@ def parse_args(args_raw):
 
     parser.add_argument("--format", choices=["csv", "json"], default="csv")
 
+    parser.add_argument("--types", nargs="+")
+
     parser.add_argument(
-        "--types",
-        nargs="+",
-        default=convert.DEFAULT_TYPES,
-        choices=convert.DEFAULT_TYPES,
+        "--all-types",
```

Comment:
Where is this parameter being used?

Reply:
Whoops! Nowhere. It's a vestige of a previous approach I considered, but decided not to pursue. Now removed in latest commit. Thanks for catching.